java/ql/src/experimental/Security/CWE/CWE-089/MyBatisCommonLib.qll

```diff
@@ -152,6 +153,8 @@ predicate isMybatisXmlOrAnnotationSqlInjection(
         unsafeExpression.matches("${param" + (i + 1) + "%}")
         or
         unsafeExpression.matches("${arg" + i + "%}")
+        or
+        unsafeExpression.regexpMatch("\\$\\{\\s*" + ma.getMethod().getParameter(i).getName() + "\\s*(,.*?)?\\s*\\}")
```

Comment:
```suggestion
        unsafeExpression.regexpMatch("\\$\\{\\s*" + ma.getMethod().getParameter(i).getName() + "\\s*(,[^}]*)?\\}")
```
Might as well be explicit (untested; I don't think the `}` inside the character class needs escaping though)

Reply:
If we're doing this here we might as well do it in the other cases too

Reply:
consider blankspace, comma and dot field, I change it to `"\\$\\{\\s*" + var + "\\b[^}]*?\\}"`
case like `${ user . name , jdbcType = VARCHAR , javaType = java.lang.String }`